test/jdk/javax/swing/JTabbedPane/TestJTabbedPaneOpaqueColor.java

```diff
@@ -0,0 +1,176 @@
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.GridLayout;
+import javax.swing.AbstractAction;
+import javax.swing.AbstractButton;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTabbedPane;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+/*
+ * @test
+ * @bug 8226990
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Test JTabbedPane's contentarea, tab area and tab color
+ *          for different LAFs when opacity is enabled or disabled.
+ * @run main/manual TestJTabbedPaneOpaqueColor
+ */
+
+public class TestJTabbedPaneOpaqueColor {
+    private static JFrame frame;
+    private static JTabbedPane tabPane;
+    private static final String INSTRUCTIONS =
+            " The background color of panel (which contains the tabbed pane)" +
+                    "is green.\n" +
+            " The background color of the tabbed pane is red.\n\n" +
+            " Check the default behaviour of the tabbed pane:\n" +
+            "    - the area behind tabs is transparent (it must be green);\n" +
+            "    - the tabs area is opaque (it must be red, except the selected" +
+                    "tab which must be gray);\n" +
+            "    - the content area is opaque(it must be gray);\n\n" +
+            " Click to the check box 'opaque'. And be sure that:\n" +
+            "    - the area behind tabs is opaque (it must be red);\n" +
+            "    - the tabs area is opaque (it must be red, except the selected" +
+                    "tab which must be gray);\n" +
+            "    - the content area is opaque (it must be gray);\n\n" +
+            " Check this behaviour for other LAFs and tab layout.\n\n" +
+            " NOTE: For Nimbus LAF, tabs color are specific to nimbus style.";
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
```

Comment:
Test needs to be updated as per the new PassFailJFrame builder changes. 

`PassFailJFrame.builder()`